config/routes.rb

```diff
@@ -3,4 +3,8 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  # get "/users" to: "users#index"
```

Comment:
- [ ] it would be better to remove the unused commented code as the previous reviewer asked in  order to keep the file clean and pro